storage/range.go

```diff
@@ -1022,8 +1022,7 @@ func (r *Range) maybeGossipFirstRange() error {
 // the initial update, and the range itself re-triggers updates following
 // writes that may have altered any of the maps.
 //
-// Note that maybeGossipConfigs does not check the leader lease; it is called
-// on only when the lease is actually held.
+// Note that maybeGossipConfigs gossips informatino only when the lease is actually held.
```

Comment:
s/informatino/information/.

Also, check out the comment just below this line.